app/constants/__init__.py

```diff
@@ -99,3 +99,7 @@
 }
 
 OPENRECORDS_DL_EMAIL = "openrecords@records.nyc.gov"
+
```

Comment:
Instead of using hardcoded IDs, add a new type to the Reasons table so that there are now CLOSING_FULFILLED, CLOSING_DENIED, DENIED